WordPress/src/test/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainsViewModelTest.kt

```diff
@@ -316,4 +362,17 @@ class SiteCreationDomainsViewModelTest {
         event.error = SuggestDomainError("GENERIC_ERROR", "test")
         return event
     }
+
+    /**
+     * Helper function that creates the current sanitized query being used to generate the domain suggestions.
+     * It returns a test domain that's based on the test suggestions being used so that the app can behave in it's
+     * normal [DomainsModelAvailableUiState] state. It also returns an unavailable domain query so that the
+     *  [DomainsModelUnavailabilityUiState] state is activated.
+     */
+    private fun createSanitizedDomainResult(isDomainAvailableInSuggestions: Boolean) =
+            if (isDomainAvailableInSuggestions) {
+                "${MULTI_RESULT_DOMAIN_FETCH_QUERY.first}-1"
```

Comment:
Let me know if I should hardcode this. I just utilized it so that there's a domain match with the first item in the list and the unavailability state wouldn't show since the domain would be available. So I think it's fine but I am more than open to more thoughts on what will be clearer for the overall codebase 🙇

Reply:
Thanks for the explanation! The usage makes sense ;)